src/main/java/com/uber/cadence/samples/shadowing/README.md

```diff
@@ -0,0 +1,37 @@
+##Shadowing example:
+
+Shadowing worker uses for detecting workflow non-deterministic error 
+prior to the workflow code deployment to prod.
+
+More detail can be found: [design doc](https://github.com/uber/cadence/blob/master/docs/design/workflow-shadowing/2547-workflow-shadowing.md)
+
+1. To run this example, start a 0.21+ cadence server.
+
+2. Run a few HelloActivity workflow to generate workflow records.
+```
+./gradlew -q execute -PmainClass=com.uber.cadence.samples.hello.HelloActivity
+```
+
+3. Run the traffic shadowing
```

Comment:
Shall we also mention the shadower for local testing in the README?

Reply:
Yes. Updated. I will also point the client doc to both samples